kiwi/runtime_checker.py

```diff
@@ -134,19 +136,51 @@ def check_docker_tool_chain_installed(self):
         for those tools to be installed in the build system and fails if
         it can't find them
         """
-        message = dedent('''\n
-            Required tool {0} not found in caller environment
+        message_tool_not_found = dedent('''\n
+            Required tool {name} not found in caller environment
 
             Creation of docker images requires the tools umoci and skopeo
             to be installed on the build system. For SUSE based systems
             you can find the tools at:
 
             http://download.opensuse.org/repositories/Virtualization:/containers
         ''')
+        message_version_unsupported = dedent('''\n
+            {name} tool found in unsupported version
+
+            Expected version: v{want_version}.x.x but got: v{got_version}.x.x
+        ''')
+
+        expected_version = 0
```

Comment:
I will coordinate with @cyphar about the version to require, but from what I understood from him, this should be fine for quite a while, as no breaking changes are expected.